tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>You don't need this section for clustering since the "training" section has nothing unique to clustering. </p><p> </p><p>For pruning, it's needed because we have the two callbacks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>OK. I deleted the training section.</p><p>BTW: I moved tips to this comprehensive guide according to https://github.com/tensorflow/model-optimization/pull/301#discussion_r405157069</p><p> </p>

Reply:
<blockquote>BTW: I moved tips to this comprehensive guide according to https://github.com/<a href="https://github.com/tensorflow/model-optimization/pull/301" target="_blank">/pull/301</a>#discussion_r405157069</blockquote><p>That makes sense</p><p> </p><p> </p><blockquote>OK. I deleted the training section.</blockquote><p>It was not deleted.</p>

Reply:
<p>Done, thanks!</p>